packages/browser/core/src/core/auth/internal/StitchAuthImpl.ts

```diff
@@ -77,6 +85,90 @@ export default class StitchAuthImpl extends CoreStitchAuth<StitchUser>
     return super.loginWithCredentialInternal(credential);
   }
 
+  public loginWithRedirect(credential: StitchRedirectCredential): void {
+    const { redirectUrl, state } = this.prepareRedirect(
+      credential,
+    );
+
+    window.location.replace(
+      this.browserAuthRoutes.getAuthProviderRedirectRoute(
+        credential,
+        redirectUrl,
+        state,
+        this.deviceInfo
+      )
+    );
+  }
+
+  public linkWithRedirect(
```

Comment:
[nit] `linkUserWithRedirectInternal` Users aren't supposed to call this directly